libraries/Microsoft.Bot.Builder.Dialogs.Adaptive/DialogManager.cs

```diff
@@ -82,6 +82,7 @@ public Dialog RootDialog
                 if (value != null)
                 {
                     this.rootDialogId = value.Id;
+                    this.Dialogs.TelemetryClient = value.TelemetryClient;
```

Comment:
> this.Dialogs.TelemetryClient = value.TelemetryClient; [](start = 19, length = 54)

?? Is this right?

Reply:
Yes, the underlying DialogSet needs to have the TelemetryClient set, after which point it will automatically assign the client to any dialogs that are added.